modules/control-utility/src/main/java/org/apache/ignite/internal/commandline/CheckpointCommand.java

```diff
@@ -62,8 +78,11 @@ public class CheckpointCommand extends AbstractCommand<Void> {
                 new VisorTaskArgument<>(nodes.stream().map(GridClientNode::nodeId).collect(Collectors.toList()), arg(), false)
             );
 
-            if (res.isSuccess())
-                log.info("Checkpointing completed successfully on " + res.numberOfSuccessNodes() + " nodes.");
+            if (res.isSuccess()) {
+                String nodeMsgPart = nodes.size() == 1 ? 1 + " node." : res.numberOfSuccessNodes() + " nodes.";
+
+                log.info("Checkpointing completed successfully on " + nodeMsgPart);
```

Comment:
Either you need to correct it on line 87 as well, or leave it as before without defining the correct ending.